### PR TITLE
Zeus w caam review

### PR DESCRIPTION
**Pull request by lee19830115**
adjust image sizes; add BG network logo; set boot mode to emmc

### CODE REVIEW

File: docs/Quick_Start_Guide.md [hunk 1]
```diff
@@ -49,16 +52,17 @@ repo sync
 
 The files and directories that are in the bgn-essa-c444
```
Comment (rlysecky) [inline docs/Quick_Start_Guide.md]:
```suggestion
Files and directories that are in the `bgn-essa-c444`:
```

File: docs/Quick_Start_Guide.md [hunk 2]
```diff
@@ -49,16 +52,17 @@ repo sync
...
 
-<p align="center">
-    <img src="assets/sync_dir.png" alt="sync_dir" />
+<p align="left">
+    <img src="assets/sync_dir.png" width=600 alt="sync_dir" />
 </p>
 
 The Yocto meta directories that are in the sources
```
Comment (rlysecky) [inline docs/Quick_Start_Guide.md]:
```suggestion
Yocto meta directories that are in `sources`:
```

File: docs/Quick_Start_Guide.md [hunk 3]
```diff
@@ -49,16 +52,17 @@ repo sync
...
 
-<p align="center">
-    <img src="assets/yocto_meta_dir.png" alt="yocto_meta_dir" />
+<p align="left">
+    <img src="assets/yocto_meta_dir.png" width=800 alt="yocto_meta_dir" />
 </p>
 
+
 #### 4. setup the build environment at first time
```
Comment (rlysecky) [inline docs/Quick_Start_Guide.md]: I think we should start each numbered step with a capital letter.

File: docs/Quick_Start_Guide.md [hunk 4]
```diff
@@ -73,8 +77,8 @@ bitbake core-image-base
 
 The following build configurations should show up in the terminal:
 
-<p align="center">
-    <img src="assets/yocto_build_config.png" alt="yocto_build_config" />
+<p align="left">
+    <img src="assets/yocto_build_config.png" width=800 alt="yocto_build_config" />
 </p>
 
 **Note**: _the image build may take a few hours at the first time_
```
Comment (rlysecky) [inline docs/Quick_Start_Guide.md]:
```suggestion
**Note**: _The initial image build may take a few hours_
```

**Review by rlysecky — COMMENTED** [review]
This guide looks great. There are few formatting inconsistencies with capitalization that would be good to update. I left a comment for the first one I noticed.

File: docs/Quick_Start_Guide.md [hunk 5]
```diff
@@ -193,85 +197,85 @@ dd if=/dev/zero of=encrypted.img bs=1M count=32
...
 ```bash
 losetup /dev/loop0 encrypted.img
 ```
 
-#### 17. use the imported key for block encryption
+#### 17. Use the imported key for block encryption
 
 ```bash
-dmsetup -v create myEncryptedBlock --table "0 $(blockdev --getsz /dev/loop0) crypt capi:tk(cbc(aes))-plain:36:logon:mykey2: 0 /dev/loop0 0 1 sector_size:512"
+dmsetup -v create myEncryptedBlock --table "0 $(blockdev --getsz /dev/loop0) crypt capi:tk(cbc(aes))-plain :36:logon:mykey2: 0 /dev/loop0 0 1 sector_size:512"
```
Comment (danie007) [inline docs/Quick_Start_Guide.md:268]: Nice catch and kudos for updating the `dmsetup` commands 👍

File: meta-mender-c444/templates/local.conf.append [hunk 6]
```diff
@@ -14,8 +14,8 @@ PREFERRED_PROVIDER_u-boot = "u-boot-imx"
 IMAGE_CLASSES += "mender-setup-c444"
 
 # Uncomment the following 2 lines to boot from eMMC on imx8mq-itx-p-c444
```
Comment (danie007) [inline meta-mender-c444/templates/local.conf.append]: Updating the boot medium to internal eMMC is a good idea 💡 . But can we update the `comment` as well?
```suggestion
# Boot from eMMC
```

File: meta-mender-c444/templates/local.conf.append [hunk 7]
```diff
@@ -14,8 +14,8 @@ PREFERRED_PROVIDER_u-boot = "u-boot-imx"
...
-# MENDER_STORAGE_DEVICE_imx8mq-itx-p-c444 = "/dev/mmcblk0"
-# MENDER_UBOOT_STORAGE_DEVICE_imx8mq-itx-p-c444 = "0"
+MENDER_STORAGE_DEVICE_imx8mq-itx-p-c444 = "/dev/mmcblk0"
+MENDER_UBOOT_STORAGE_DEVICE_imx8mq-itx-p-c444 = "0"
 
 # Boot from SD
```
Comment (danie007) [inline meta-mender-c444/templates/local.conf.append]:
```suggestion
# Uncomment the following line to boot from SD Card on imx8mq-itx-p-c444
```

**Review by danie007 — CHANGES_REQUESTED** [review]
Over all nice work 👌, some minor changes can be made, if make sense!

---

CC @tkmozhi

File: meta-mender-c444/templates/local.conf.append (hunk 6 above)
Comment (lee19830115) [reply meta-mender-c444/templates/local.conf.append]: look nice. maybe we reword the comments in this way?

SD and eMMC boot are mutually exclusive. Please select only one, either SD boot or eMMC boot

Boot from eMMC
Uncomment the following 2 lines to boot from eMMC on imx8mq-itx-p-c444


Boot from SD card
Uncomment the following line to boot from a SD card on imx8mq-itx-p-c444

File: meta-mender-c444/templates/local.conf.append [hunk 8]
```diff
@@ -13,9 +13,13 @@ MENDER_DEVICE_TYPES_COMPATIBLE = "imx8mq-itx-p-c444"
 PREFERRED_PROVIDER_u-boot = "u-boot-imx"
 IMAGE_CLASSES += "mender-setup-c444"
 
+# SD and eMMC boot are mutually exclusive. Please select only one, either SD boot or eMMC boot
+
+# Boot from eMMC
 # Uncomment the following 2 lines to boot from eMMC on imx8mq-itx-p-c444
```
Comment (danie007) [inline meta-mender-c444/templates/local.conf.append]: eMMC is already in effect (_i.e., uncommented_). Hence the following comment/line shall be removed 

```suggestion
```

—-

CC @tkmozhi

**Review by danie007 — APPROVED** [review]
LGTM 👍

**Pull request merged**